settings.gradle

```diff
@@ -144,39 +144,39 @@ include ':instrumentation:mongo:mongo-3.7'
 include ':instrumentation:mongo:mongo-async-3.3'
 include ':instrumentation:mongo:mongo-common'
 include ':instrumentation:mongo:mongo-testing'
-include ':instrumentation:netty:netty-3.8'
-include ':instrumentation:netty:netty-4.0'
-include ':instrumentation:netty:netty-4.1'
-include ':instrumentation:okhttp:okhttp-2.2'
-include ':instrumentation:okhttp:okhttp-3.0'
-include ':instrumentation:opentelemetry-api-beta'
+include ':instrumentation:netty:netty-3.8:javaagent'
+include ':instrumentation:netty:netty-4.0:javaagent'
+include ':instrumentation:netty:netty-4.1:javaagent'
+include ':instrumentation:okhttp:okhttp-2.2:javaagent'
+include ':instrumentation:okhttp:okhttp-3.0:javaagent'
+include ':instrumentation:opentelemetry-api-beta:javaagent'
 include ':instrumentation:oshi:javaagent'
 include ':instrumentation:oshi:library'
-include ':instrumentation:play:play-2.3'
-include ':instrumentation:play:play-2.4'
-include ':instrumentation:play:play-2.6'
-include ':instrumentation:play-ws:play-ws-1.0'
-include ':instrumentation:play-ws:play-ws-2.0'
-include ':instrumentation:play-ws:play-ws-2.1'
-include ':instrumentation:play-ws:play-ws-common'
+include ':instrumentation:play:play-2.3:javaagent'
+include ':instrumentation:play:play-2.4:javaagent'
+include ':instrumentation:play:play-2.6:javaagent'
+include ':instrumentation:play-ws:play-ws-1.0:javaagent'
+include ':instrumentation:play-ws:play-ws-2.0:javaagent'
+include ':instrumentation:play-ws:play-ws-2.1:javaagent'
+include ':instrumentation:play-ws:play-ws-common:javaagent'
 include ':instrumentation:play-ws:play-ws-testing'
-include ':instrumentation:rabbitmq-2.7'
-include ':instrumentation:ratpack-1.4'
-include ':instrumentation:reactor-3.1'
-include ':instrumentation:reactor-netty-0.9'
-include ':instrumentation:rediscala-1.8'
-include ':instrumentation:redisson-3.0'
-include ':instrumentation:rmi'
-include ':instrumentation:rxjava-1.0'
-include ':instrumentation:rxjava-1.0'
-include ':instrumentation:scala-concurrent'
+include ':instrumentation:rabbitmq-2.7:javaagent'
+include ':instrumentation:ratpack-1.4:javaagent'
+include ':instrumentation:reactor-3.1:javaagent'
+include ':instrumentation:reactor-netty-0.9:javaagent'
+include ':instrumentation:rediscala-1.8:javaagent'
+include ':instrumentation:redisson-3.0:javaagent'
+include ':instrumentation:rmi:javaagent'
+include ':instrumentation:rxjava-1.0:javaagent'
+include ':instrumentation:rxjava-1.0:javaagent'
```

Comment:
`rxjava-1.0` looks more like a library instrumentation, should we add javaagent here?

Reply:
great point, sent a separate PR for this for visibility #1732